### PR TITLE
News html

### LINKED ISSUE #26: Restructure newspage html
- Page currently uses a table body, how ever incorporating more divs may be more effective and simpler. 
- Should format for user as a simple news page. 
- Format divs w/ text to the right of page.

### PR DESCRIPTION
**Pull request by nburd90**
- Created div structure for html. 
- Will edit positioning and layout in #22.
- Will added further html if needed.

### CODE REVIEW

File: news.php [hunk 1]
```diff
@@ -8,96 +8,76 @@
...
-                    <p>London, 30th Nov 2020 Independent has widely sold the SXSW winning documentary THE DONUT KING,
-                        with VICE acquiring the film in a multi-territory deal.
-                    </p>  
-                </p>              
+            <div class="newsSection">  
+                <!--<div width:23%; position: relative;>
+                <p class="header-medium">2020</p>
                 </div>
-                <a href="https://www.independentfilmcompany.com/news/independent_announces_sales_on_documentary_the_donut_king__led_by_a_multiterritory_acquisition_from_vice_media.html"
+                <div class="news-image-holder" style="background:url(crispy-images/newspic2.jpeg)"></div>-->
```
Comment (billythakidd04) [inline news.php]: Remove the commented code if its not getting used

File: news.php [hunk 2]
```diff
@@ -8,96 +8,76 @@
...
-                <p class="news-date">14 October 2020</p>
-                <div class="newsItemSummary">
-                    <p>
-                    Vertigo Releasing has acquired UK and Ireland distribution rights from Independent to Kieron J. Walshs cycling drama 
-                    <em>The Racer</em>
-                    </p>
-                </p>
                 </div>
-                <a href="https://www.independentfilmcompany.com/news/vertigo_releasing_picks_up_cycling_drama_the_racer_for_uk__ireland.html"
+                <div class="news-image-holder" style="background: url(crispy-images/newspic3.jpg);"></div>-->
```
Comment (billythakidd04) [inline news.php]: deleted commented code

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
Kill the extraneous commented code if its not getting used.

File: news.php [hunk 3]
```diff
@@ -8,96 +8,74 @@
...
-                    <p class="news-date">09 February 2021</p>
+    <div class="container">
+        <div class="newsPage">
+            <h1>News</h1>
+            <div class="newsSection">
+                    <p class="header-medium">2021</p>
+                    <div class="news-image-holder">
+                        <img src="newspic3.jpg">
+                    </div>
+                <div class="newsItem">
```
Comment (billythakidd04) [inline news.php:19]: fix your whitespace formatting here

File: news.php [hunk 4]
```diff
@@ -8,96 +8,74 @@
...
-                <p class="news-date">30 November 2020</p>
-                <div class="newsItemSummary">
-                    <p>London, 30th Nov 2020 Independent has widely sold the SXSW winning documentary THE DONUT KING,
-                        with VICE acquiring the film in a multi-territory deal.
-                    </p>  
-                </p>              
+                    <div class="news-image-holder">
+                       <img src="newspic2.jpg"> 
+                    </div>
+                <div class="newsItem">
```
Comment (billythakidd04) [inline news.php:43]: formatting

**Review by billythakidd04 — CHANGES_REQUESTED** [review]

File: news.php (hunk 3 above)
Comment (nburd90) [reply news.php:19]: 8b3bf4c

File: news.php (hunk 4 above)
Comment (nburd90) [reply news.php:43]: 8b3bf4c

File: news.php [hunk 5]
```diff
@@ -13,9 +13,9 @@
             <h1>News</h1>
             <div class="newsSection">
                     <p class="header-medium">2021</p>
```
Comment (billythakidd04) [inline news.php:15]: ok now the p tag is off... did you run an auto format?

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
if this is giving you trouble text me. formatting is extremely important for readability. on the job, you won't be the last person to touch your code. its important to leave it so _others_ can read it easily.

**Review by billythakidd04 — APPROVED** [review]

**Pull request merged**